### PR TITLE
Backfilling 2 cruises that were missed from this summer

### PR DESCRIPTION
**Pull request by cdobs**
Backfilled Cruiseinformation with 2 glider/AUV cruises:
-Pioneer 12c (QLR 3204-01210)
-MZ20190727 for cp_379 recovery only on the F/Z Mattanza. This vessel was not in the shiplist.csv so I added it there

### CODE REVIEW

File: vessel/shiplist.csv [hunk 1]
```diff
@@ -28,7 +28,8 @@ KM,R/V,Kilo Moana,33KB,Hawaii,WDA7827,369565000,9229037,56.7,7.7,7.6,AGOR-26,Y,Y
 KN,R/V,Knorr,316N,WHOI,KCEJ,367246000,7738618,85,7.2,6,AGOR-15,(N),Y
 LMG,R/V,Laurence M. Gould,33LG,USAP,WCX7445,368138000,9137337,70.1,5.8,5.7,,Y,Y
 MGL,R/V,Marcus G. Langseth,33H3,LDEO,WDC6698,367059880,9010137,71.6,6.7,6.7,,Y,Y
-MM,R/V,Maria S. Merian,,IOW,,�211753000,9274197,,,,,Y,N
+MM,R/V,Maria S. Merian,,IOW,,�211753000,9274197,,,,,Y,N
+MZ,F/V,Mattanza,,TM,,,,,,,,Y,N
 EW,R/V,Maurice Ewing,3230,LDEO,WLDZ,366167000,8203153,70.1,6.7,5.3,,(N),Y
```
Comment (snwhite) [inline vessel/shiplist.csv]: Looks good (SNW)

File: cruise/CruiseInformation.csv [hunk 2]
```diff
@@ -168,6 +168,8 @@ ML20190612,R/V Miss Linda,2019-06-12T13:00:00,2019-06-13T18:00:00,Endurance glid
 WJ20190617,M/V Warren Jr.,2019-06-17T18:00:00,2019-06-20T00:04:00,Pioneer 12b glider and AUV deployment
 EK20190624,R/V Elakha,2019-06-24T14:55:00,2019-06-24T19:00:00,Endurance ce02shsp_00013 deployment
 EK20190703,R/V Elakha,2019-07-03T15:17:00,2019-07-03T21:00:00,Endurance ce02shsp_00014 glider deployment
+WJ20190709,M/V Warren Jr.,2019-07-09T09:00:00,2019-07-12T19:00:00,Pioneer 12c AUV deployment
+MZ20190727,F/V Mattanza,2019-07-27T04:00:00,2019-07-27T23:00:00,Pioneer glider cp_379 recovery
 AR35-05,R/V Neil Armstrong,2019-08-02T14:00:00,2019-08-24T14:25:00,Irminger 6
```
Comment (snwhite) [inline cruise/CruiseInformation.csv]: Looks good (SNW)

Comment (reedan88) [reply cruise/CruiseInformation.csv]: Looks good.

File: vessel/shiplist.csv (hunk 1 above)
Comment (reedan88) [reply vessel/shiplist.csv]: Looks good as well.

**Pull request merged**